### PR TITLE
[zh-cn] Sync web-ui-dashboard.md

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
Sync `content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md` with en page.

### CODE REVIEW

File: content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 1]
```diff
@@ -14,6 +15,8 @@ reviewers:
 - shu-mutou
 - mikedanese
 title: Deploy and Access the Kubernetes Dashboard
+description: >-
+  Deploy the web UI (Kubernetes Dashboard) and access it.
 content_type: concept
 weight: 10
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md:21]:
```suggestion
weight: 30
```

File: content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ card:
   name: tasks
   weight: 30
   title: 使用 Web 界面 Dashboard
+  description: 部署 Web 界面（Kubernetes 仪表板）并访问它
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md:9]:
```suggestion
  description: 部署并访问 Web 界面（Kubernetes 仪表板）。
```

File: content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md (hunk 1 above)
Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md:21]: What is the reason for such a change？Emm...

Comment (windsonsea) [reply content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md:21]: Your title is to `sync` but the current zh has a different weight from the commented weight.

Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md:21]: EN section as follows:
```md
reviewers:
- floreks
- maciaszczykm
- shu-mutou
- mikedanese
title: Deploy and Access the Kubernetes Dashboard
description: >-
  Deploy the web UI (Kubernetes Dashboard) and access it.
content_type: concept
weight: 10
card:
  name: tasks
  weight: 30
  title: Use the Web UI Dashboard
```

CN section as follows:
```md
title: 部署和访问 Kubernetes 仪表板（Dashboard）
content_type: concept
weight: 10
card:
  name: tasks
  weight: 30
  title: 使用 Web 界面 Dashboard
  description: 部署并访问 Web 界面（Kubernetes 仪表板）。
```

I am sorry, do I miss something? Or  you missed up `weight` with `card.weight`?

Comment (windsonsea) [reply content/zh-cn/docs/tasks/access-application-cluster/web-ui-dashboard.md:21]: oh oh, sorry, you are correct! -_!

**Pull request merged**